redis/client.py

```diff
@@ -525,23 +541,50 @@ def _disconnect_raise(self, conn, error):
         ):
             raise error
 
+    def get_from_local_cache(self, command):
+        if (
+            self.client_cache is None
+            or command[0] in self.cache_blacklist
+            or command[0] not in self.cache_whitelist
+        ):
+            return None
+        return self.client_cache.get(command)
+
+    def add_to_local_cache(self, command, response, keys):
+        if (
+            self.client_cache is not None
+            and (self.cache_blacklist == [] or command[0] not in self.cache_blacklist)
+            and (self.cache_whitelist == [] or command[0] in self.cache_whitelist)
+        ):
+            self.client_cache.set(command, response, keys)
+
+    def delete_from_local_cache(self, command):
+        if self.client_cache is not None:
+            self.client_cache.delete(command)
```

Comment:
Is it cheaper to attempt a deletion whether or not the field exists - and just trigger the delete?